synapse/telepath.py

```diff
@@ -852,7 +854,10 @@ async def _teleLinkLoop(self):
                 raise
 
             except Exception as e:
-                logger.warning(f'telepath client ({s_urlhelp.sanitizeUrl(url)}): {e}')
+                now = time.monotonic()
+                if now > lastlog + 60.0:  # don't logspam the disconnect message more than 1/min
```

Comment:
Didn't we plumb a utilitized log-spam reducer concept at some point?

Reply:
Newp.